gcloud/bigquery/client.py

```diff
@@ -176,8 +174,7 @@ def list_jobs(self, max_results=None, page_token=None, all_users=None,
         path = '/projects/%s/jobs' % (self.project,)
         resp = self.connection.api_request(method='GET', path=path,
                                            query_params=params)
-        jobs = [self._job_from_resource(resource)
-                for resource in resp['jobs']]
+        jobs = [self.job_from_resource(resource) for resource in resp['jobs']]
```

Comment:
I don't know:  it is hard to figure out how defensive we need to be against the back-end violating its documented response formats.  I've tended to write the implementations assuming that the docs are true, and then fix them up only when real-world usage shows that belief to be naive.